systemtests/src/test/java/io/enmasse/systemtest/brokered/SmokeTest.java

```diff
@@ -6,11 +6,11 @@
 
 import io.enmasse.systemtest.*;
 import io.enmasse.systemtest.ability.ITestBaseBrokered;
+import io.enmasse.systemtest.ability.ITestSeparator;
```

Comment:
this import is useless, please use idea autoformat before commit/push

Reply:
removed